WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -1384,20 +1361,20 @@ - (void)uploadFeaturedMedia:(Media *)media
 
 #pragma mark - WPTableImageSourceDelegate
 
-- (void)tableImageSource:(WPTableImageSource *)tableImageSource
-              imageReady:(UIImage *)image
-            forIndexPath:(NSIndexPath *)indexPath
-{
-    self.featuredImage = image;
-    [self.tableView reloadData];
-}
-
-- (void)tableImageSource:(WPTableImageSource *)tableImageSource
- imageFailedforIndexPath:(NSIndexPath *)indexPath
-                   error:(NSError *)error
-{
-    [self featuredImageFailedLoading:indexPath withError:error];
-}
+//- (void)tableImageSource:(WPTableImageSource *)tableImageSource
+//              imageReady:(UIImage *)image
+//            forIndexPath:(NSIndexPath *)indexPath
+//{
+//    self.featuredImage = image;
+//    [self.tableView reloadData];
+//}
+//
+//- (void)tableImageSource:(WPTableImageSource *)tableImageSource
+// imageFailedforIndexPath:(NSIndexPath *)indexPath
+//                   error:(NSError *)error
+//{
+//    [self featuredImageFailedLoading:indexPath withError:error];
+//}
```

Comment:
@SergioEstevao Were you intending to remove this commented block?

Reply:
I need to delete it.